config/ecrpublic/config.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ecrpublic
+
+import (
+	"github.com/crossplane-contrib/terrajet/pkg/config"
+)
+
+// Configure adds configurations for ecrpublic group.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("aws_ecrpublic_repository", func(r *config.Resource) {
```

Comment:
similarly, should be use async here: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecrpublic_repository#delete